### PR TITLE
Exam Data Model Fix for HTTP Method Testing

### PR DESCRIPTION
**Pull request by akroon3r**
During the testing of http methods on the front end, it was found that specific fields on the exam model were required to be nullable due to government workflow practices. Also found that the room_id field on the exams model was an error/typo, so this field was completely removed from the exam model/schema. This PR required a new database migration, and therefore was tested to ensure that bootstrap data didn't cause any fatal errors prior to commit.

### CODE REVIEW

File: api/app/models/bookings/exam.py [hunk 1]
```diff
@@ -21,18 +21,17 @@ class Exam(Base):
     exam_id = db.Column(db.Integer, primary_key=True, autoincrement=True, nullable=False)
     booking_id = db.Column(db.Integer, db.ForeignKey("booking.booking_id", ondelete="set null"), nullable=True)
     exam_type_id = db.Column(db.Integer, db.ForeignKey("examtype.exam_type_id"), nullable=False)
-    invigilator_id = db.Column(db.Integer, db.ForeignKey("invigilator.invigilator_id"), nullable=False)
+    invigilator_id = db.Column(db.Integer, db.ForeignKey("invigilator.invigilator_id"), nullable=True)
     office_id = db.Column(db.Integer, db.ForeignKey("office.office_id"), nullable=False)
-    room_id = db.Column(db.Integer, db.ForeignKey("room.room_id"), nullable=False)
     event_id = db.Column(db.String(25), nullable=False)
     exam_name = db.Column(db.String(50), nullable=False)
     examinee_name = db.Column(db.String(50), nullable=False)
```
Comment (sjrumsby) [inline api/app/models/bookings/exam.py]: Should this be nullable?

Comment (sjrumsby) [reply api/app/models/bookings/exam.py]: From Slack, this should be nullable.

Comment (akroon3r) [reply api/app/models/bookings/exam.py]: This will be nullable.

Comment (akroon3r) [reply api/app/models/bookings/exam.py]: This is now nullable.

**Comment by akroon3r** [comment]
@sjrumsby please re-review.

**Pull request merged**